src/main/java/seedu/bigpp/parser/Parser.java

```diff
@@ -11,6 +11,10 @@
 public class Parser {
 
     public Command parseCommand(String userInput) {
+        if (userInput.equals("back")) {
+            return new BackCommand();
+        }
+
```

Comment:
followed your suggestion of checking for back command since it exists throught all menus